test/benchmarks/sails.load.test.js

```diff
@@ -205,10 +206,6 @@ function setupBenchmarks() {
  * @this {Array} benchmarks
  */
 function reportBenchmarks() {
-  var _ = require('lodash');
```

Comment:
Can't tell from the PR alone, so just checking:  did you mean to remove this?  i.e. do you mean it is an unused dependency?